source_gen/lib/src/generated_output.dart

```diff
@@ -2,29 +2,32 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'dart:convert';
-
 import 'generator.dart';
 
 class GeneratedOutput {
   final String output;
   final Generator generator;
+
+  @Deprecated('Always null. Will be removed in the next release.')
```

Comment:
This class is not in the public API, we don't need to worry about deprecating.

I strongly suspect we can drop this class entirely and yield the `String` directly.

Reply:
Yup. See follow-up commit

Reply:
Still should remove the class. At this point it is only a wrapper around the `String output`.

Reply:
Oh...hrm...k

Reply:
done!

Reply:
Hand to revert that last change, @natebosch – we use the generator name in the output.

See my latest commit. I just store the name instead of holding on to the generator.

Reply:
SGTM